src/pages/OnboardingPage/EndScreen.js

```diff
@@ -1,131 +1,173 @@
+		url: 'https://opensource.appbase.io/reactivesearch',
+	},
+	{
+		name: 'Vue',
+		image1x: '/static/images/onboarding/finish-screen/vue@1x.png',
+		image2x: '/static/images/onboarding/finish-screen/vue@2x.png',
+		description: 'Vue UI components for building data-driven search experiences',
+		url: 'https://opensource.appbase.io/reactivesearch/vue',
+	},
+	{
+		name: 'React Native',
+		image1x: '/static/images/onboarding/finish-screen/ReactNative@1x.png',
+		image2x: '/static/images/onboarding/finish-screen/ReactNative@2x.png',
+		description: 'Build data-driven search experiences for iOS and Android apps',
+		url: 'https://opensource.appbase.io/reactivesearch/native',
+	},
+	{
+		name: 'REST APIs',
+		image1x: '/static/images/onboarding/finish-screen/REST@1x.png',
+		image2x: '/static/images/onboarding/finish-screen/REST@2x.png',
+		description: 'API for working with elasticsearch / appbase apps',
+		url: 'https://docs.appbase.io/rest-quickstart.html',
+	},
+	{
+		name: 'Javascript Library',
+		image1x: '/static/images/onboarding/finish-screen/JS@1x.png',
+		image2x: '/static/images/onboarding/finish-screen/JS@2x.png',
+		description: 'Javascript client library for working with elasticseach / appbase apps',
+		url: 'https://docs.appbase.io/javascript/quickstart.html',
+	},
+];
 
-						<div>
-							<div className="col">
-								<img
-									src="/static/images/onboarding/finish-screen/Trophy.png"
-									srcSet="/static/images/onboarding/finish-screen/Trophy.png 245w, /static/images/onboarding/finish-screen/Trophy@2x.png 490w"
-									alt="Trophy"
-								/>
-								<p>
-									You
-									{"'"}
-									ve finished the tutorial.
-								</p>
+const getFilteredList = (query) => {
+	const matchQuery = query.toLowerCase();
+	return integrations.filter(item => item.name.toLowerCase().indexOf(matchQuery) > -1);
+};
+
+class EndScreen extends React.PureComponent {
+	state = {
+		searchQuery: '',
+	};
+
+	handleSearchChange = (e) => {
+		const {
+			target: { value },
+		} = e;
+		this.setState({
+			searchQuery: value,
+		});
+	};
+
+	render() {
+		const { searchQuery } = this.state;
+		const integrationsList = searchQuery.trim() ? getFilteredList(searchQuery) : integrations;
+		const { currentApp } = this.props;
+		return (
+			<Layout>
+				<FullHeader />
+				<div className={endScreenStyles}>
+					<div className="container">
+						<div className="banner-row">
+							<div className="big-card">
+								<div className="img-container">
+									<img
+										src="/static/images/onboarding/finish-screen/SearchSandbox.png"
+										alt="search-sandobx"
+									/>
+								</div>
+								<div className="text">
+									<div>
+										<h3>SEARCH PREVIEW</h3>
+										<h1>Keep tunning search settings</h1>
+										<p>
+											Continue tuning search, filter and layout options for
+											this app.
+										</p>
+									</div>
+									<Link to={`/app/${currentApp}/search-preview`}>
```

Comment:
Also the layout is breaking in Mobile screen

Reply:
Previously, we were using:

```js
href={`/app/${window.location.search.split('=')[1]}`}
```

Reply:
@metagrover url has app number in query param `/tutorial/finish/?app=122` because of which have to find appname from redux store to link it

Reply:
Good find @jyash97!

I suggest the following: Ideally, this page is only opened after following the tutorial, so there is a high likelihood of having the app query string param. However, in the case there isn't one:

* We can disable the Search Preview button,
* For dashboard, instead redirect to the main dashboard screen.

Reply:
Thanks @siddharthlatest 👍 
@jyash97 have fixed layout for mobile screen in following PR
https://github.com/appbaseio/dashboard/pull/133/commits/050053b24c1234443838e828051268814854b63d

Reply:
@lakhansamani you can easily update it to use `appName` instead of `appId` in the onboarding footer component which redirects it to the end-screen here.

Reply:
(or) better call `setCurrentApp` action to set the relevant app (This would be the ideal solution in my opinion).